### ISSUE TITLE
Pursue the “date rendering” problem on indyscan.io

### ISSUE THREAD

**Issue by swcurran** [sovrin-foundation/sovrin#332]

**Comment by swcurran** [comment]
Not quite sure what to do with this, but figured I’d add it.

**Comment by WadeBarnes** [comment]
@lineko, You had mentioned you may have found the issue, is that correct?

**Comment by lineko** [comment]
Yes, I'm pretty sure of it but have not had time to install and test it yet.

**Comment by lineko** [comment]
I think the problem is in TxPreview.js

Two things very quickly:

1. using local time instead of UTC
2. format should be ('dd MMMM YYYY, HH:mm:ss a) instead of ('do MMMM YYYY, HH:mm:ss a)
